### PR TITLE
Feature/andrew

### PR DESCRIPTION
**Pull request by twsandrew**

### CODE REVIEW

File: tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml [hunk 1]
```diff
@@ -4,13 +4,11 @@
 #@ if constraint.to_generate():
 name: #@ constraint.constraint_name()
 resource_types: 
-- container.googleapis.com/Cluster
-- container.googleapis.com/NodePool 
-condition: 1 == 0
+- compute.googleapis.com/Network
+condition: resource.internalIpv6Range.contains ('Enabled') == false
```
Comment (vannicktrinquier) [inline tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml]: Do we have any information mentioning that "Enabled" needs to be used ?

File: tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml [hunk 2]
```diff
@@ -4,13 +4,11 @@
 #@ if constraint.to_generate():
 name: #@ constraint.constraint_name()
 resource_types: 
-- container.googleapis.com/Cluster
-- container.googleapis.com/NodePool 
-condition: 1 == 0
+- compute.googleapis.com/Network 
+condition: resource.routingConfig.routingMode.contains ('Global') == false
```
Comment (vannicktrinquier) [inline tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml]: Any documentation on that ?

Comment (onghongzheng) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml]: We followed this documentation for the resource https://cloud.google.com/vpc/docs/custom-constraints#gcloud and took inspiration from other examples of custom organisation policies to require "Global" for VPC.

File: tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml (hunk 1 above)
Comment (onghongzheng) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml]: No, we do not have any information that "Enabled" needs to be used. Same as previous example

Comment (vannicktrinquier) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml]: From where this Enabled is coming from ? Any screenshot ?

File: tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml (hunk 2 above)
Comment (vannicktrinquier) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml]: From where is Global is coming from ? Any screenshot ?

Comment (onghongzheng) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireVpcGlobalRoutingMode.yaml]: We go inspiration from this constraint and decide to try it for VPC. 
 Screenshot 2024-04-02 9 44 50 AM

File: tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml (hunk 1 above)
Comment (onghongzheng) [reply tools/custom-organization-policy-library/build/custom-constraints/network/networkRequireInternalIpv6Ranges.yaml]: Same as the previous one, took inspiration from this custom constraint.   
Screenshot 2024-04-02 9 44 50 AM

**Review by vannicktrinquier — APPROVED** [review]

**Pull request merged**